spec/sw_mapper_spec.rb

```diff
@@ -345,32 +341,29 @@
     end
   end
 
-  describe 'display_type' do
-    it 'is the display_type from the identityMetadata if it exists' do
-      skip("need to write test")
-    end
-    context 'is based upon the content type if no display_type' do
-      it 'is book when content type is book' do
-        skip("need to write test")
-      end
-      it 'is image when the content type is image, manuscript or map' do
-        skip("need to write test")
-      end
-      it 'is file when content type is not book, image, manuscript, or map' do
-        skip("need to write test")
-      end
-    end
-  end
-
   describe '#file_ids' do
-    it 'includes image_ids if display_type is image' do
-      skip("need to write test")
-    end
-    it 'includes file_ids if display_type is file' do
-      skip("need to write test")
+    let(:fake_druid) { 'zz999zz9999' }
+    let(:purl_xml_model) { double('DiscoveryIndexer::InputXml::PurlxmlModel').as_null_object }
+    let(:mapper) { described_class.new(fake_druid) }
+    it 'includes image_ids if dor_content_type is book, image, manuscript, or map' do
+      allow(purl_xml_model).to receive(:is_collection).and_return(false)
+      allow(purl_xml_model).to receive(:image_ids).and_return('zz999zz9999%2Fa24.jp2')
```

Comment:
I think this test is hiding a problem.  The test is good, but the actual image_ids from the purl_xml_model won't include druid%2F -- that fix has to be added (and probably should be a separate test).